engine/executor/dm/worker.go

```diff
@@ -165,9 +174,15 @@ func (w *dmWorker) Tick(ctx context.Context) error {
 	// update unit status periodically to update metrics
 	w.unitHolder.CheckAndUpdateStatus()
 	w.discardResource4Syncer(ctx)
+	w.DoTick(ctx)
 	return w.messageAgent.Tick(ctx)
 }
 
+func (w *dmWorker) TickImpl(ctx context.Context) error {
+	status := w.workerStatus(ctx)
+	return w.UpdateStatus(ctx, status)
```

Comment:
this is to avoid UpdateStatus message is dropped so ExtBytes always empty?
Can we change the logic of `tryUpdateStatus`?

Reply:
1. Yes
2. I think we don't need change tryUpdateStatus, still update it when stage changed

Reply:
OK we can separate trigger actions and periodical actions, but the name of `tryUpdateStatus` can be changed to `updateStatusWhenStageChange` or something.

please help update the comment, now OnWorkerOffline can appear without OnWorkerOnline https://github.com/pingcap/tiflow/blob/f7b84c7683f31f7b856d2c65bec5ec1b53c6065b/engine/framework/master.go#L107